fastlane/Fastfile

```diff
@@ -97,402 +95,407 @@ lane :test do | options |
+#
+# The process is not automated via the build_and_upload_beta lane due to
+# issues with Sparkle and code signing.
+lane :upload_app_to_appcenter do |options|
+  app_path = options[:app_path]
+  UI.user_error! "Missing .app path option.\nUsage:\n\tfastlane upload_app_to_appcenter app_path:path/to/.app" if app_path.nil?
+
+  app_path = File.expand_path(app_path)
+  UI.user_error! "Could not find .app at #{app_path}" unless File.exist?(app_path)
 
-    # Convert the UTF-16 encoded strings file to UTF-8.
-    strings_file_path = File.join(en_lproj_path, 'Localizable.strings')
-    utf8_strings = File.read(strings_file_path, encoding: 'UTF-16', mode: 'rb')
-    File.write(strings_file_path, utf8_strings, mode: 'w', encoding: 'UTF-8')
+  Tempfile.create do |t|
+    release_notes_path = File.path(t)
+    system "pushd ..; ruby ./Scripts/extract_release_notes.rb -k > #{release_notes_path}; popd"
 
-    git_commit(
-      path: en_lproj_path,
-      message: 'Freeze strings for localization',
-      allow_nothing_to_commit: true
+    appcenter_upload(
+      api_token: get_required_env('APPCENTER_API_TOKEN'),
+      owner_name: 'automattic',
+      owner_type: 'organization',
+      app_name: 'Simplenote-OSX',
+      ipa: app_path,
+      release_notes: File.read(release_notes_path),
+      notify_testers: false
     )
   end
 
-  desc 'Download strings from from Glotpress'
-  lane :download_strings_files_from_glotpress do
-    locale_codes = {
-      'ar' => 'ar',             # Arabic
-      'cy' => 'cy',             # Welsh
-      'de' => 'de',             # German
-      'el' => 'el',             # Greek
-      'es' => 'es',             # Spanish
-      'fa' => 'fa',             # Persian
-      'fr' => 'fr',             # French
-      'he' => 'he',             # Hebrew
-      'id' => 'id',             # Indonesian
-      'it' => 'it',             # Italian
-      'ja' => 'ja',             # Japanese
-      'ko' => 'ko',             # Korean
-      'nl' => 'nl',             # Dutch
-      'pt-br' => 'pt-BR',       # Portuguese (Brazil)
-      'ru' => 'ru',             # Russian
-      'sv' => 'sv',             # Swedish
-      'tr' => 'tr',             # Turkish
-      'zh-cn' => 'zh-Hans-CN',  # Chinese (China)
-      'zh-tw' => 'zh-Hant-TW'   # Chinese (Taiwan)
-    }.freeze
-    ios_download_strings_files_from_glotpress(
-      project_url: 'https://translate.wordpress.com/projects/simplenote/macos/',
-      locales: locale_codes,
-      download_dir: File.join(PROJECT_FOLDER, 'Simplenote')
-    )
+  # Rather than taking another manual step and finding the path to the
+  # archive, let's just bank on the fact that one wouldn't do many archives
+  # of the app on a given day and upload all of the archives done today.
+  Dir["#{Dir.home}/Library/Developer/Xcode/Archives/#{Time.now.strftime '%Y-%m-%d'}/Simplenote *"].each do |path|
+    system "sentry-cli upload-dif --org a8c --project simplenote-macos #{Shellwords.shellescape(path)}"
```

Comment:
💭  Another idea for a future PR / improvement: we should use the `sentry` fastlane plugin and its `upload_symbols_to_sentry` action instead at some point.

Reply:
Yep, there are a lot of things in here that I would to get upgraded at some point 🙂